staging/src/k8s.io/kubectl/pkg/cmd/cmd.go

```diff
@@ -146,7 +142,7 @@ func NewDefaultKubectlCommandWithArgs(o KubectlOptions) *cobra.Command {
 			if cmdutil.CmdPluginAsSubcommand.IsEnabled() {
 				// Command exists(e.g. kubectl create), but it is not certain that
 				// subcommand also exists (e.g. kubectl create networkpolicy)
-				if _, ok := allowedCmdsSubcommandPlugin[foundCmd.Name()]; ok {
+				if _, ok := GetAllowedCmdsAsSubcommandPlugins()[foundCmd.Name()]; ok {
```

Comment:
In general, I think it is good to avoid needless initialization, although this is pretty small and a drop in the bucket compared to all the other needless initialization that occurs, but I like the idea behind this change.

One suggestion though....

If this is going to be a function, can it just be simplified to something like:

```
if isSubcommandPluginAllowed(foundCmd) {
    // Do the things...
}
```

Reply:
Thanks @brianpursley for your review and suggestion. I changed the code in this way.